src/table/TableWrapper.jsx

```diff
@@ -85,7 +85,7 @@ export default function TableWrapper(props) {
       onKeyDown={(evt) => updatePage(evt, size.qcy, page, rowsPerPage, handleChangePage, setShouldRefocus)}
     >
       <TableContainer ref={tableSection} className={classes[containerMode]}>
-        <Table stickyHeader aria-label={`showing ${rows.length + 1} rows and ${columns.length} columns`}>
+        <Table stickyHeader role="table" aria-label={`showing ${rows.length + 1} rows and ${columns.length} columns`}>
```

Comment:
This is a mui table, don't think you can add a role like that, and it shouldn't be necessary

Reply:
✅ fixed